src/Carbon/Carbon.php

```diff
@@ -1209,6 +1209,25 @@ public static function getWeekendDays()
         return static::$weekendDays;
     }
 
+    /**
+     * Get the all dates of week
+     *
+     * @return array
+     */
+    public static function getCurrentWeekDays()
+    {
+        $startOfWeek = Carbon::instance((new static)->startOfWeek());
+
+        $weekDays = [$startOfWeek->toDateTimeString()];
+
+        for ($i = 1; $i < 7; $i++)
```

Comment:
Why not just loop from 0 to < DAYS_PER_WEEK, no need to inject the first day of the week with a special case.